src/driver/mysql/MysqlDriver.ts

```diff
@@ -478,42 +478,50 @@ export class MysqlDriver implements Driver {
     /**
      * Prepares given value to a value to be persisted, based on its column type or metadata.
      */
-    prepareHydratedValue(value: any, columnMetadata: ColumnMetadata): any {
-        if (value === null || value === undefined)
-            return columnMetadata.transformer ? ApplyValueTransformers.transformFrom(columnMetadata.transformer, value) : value;
+    prepareHydratedValue(value: any, columnMetadata: ColumnMetadata | string): any {
```

Comment:
you have changed only one driver. In oder to deliver such a big feature we need to support it across all drivers.

Reply:
For now I can only change it in this controller. But I think that it's better to offer it in one at least than to continue without offering it in any.

Reply:
The problem is what happens if we accept it for just MySQL? Those developers start using this syntax, later when we go to add it to other drivers and realize it needs a different kind of syntax we have to make a breaking change for MySQL users.

Reply:
And what is the alternative? Continue without offering this basic functionality without a plan to implement it? Any framework has breaking changes and it is not a problem. For this, there are migration guides between versions...

Reply:
I am working on improving the overall functionality of the query builder and have some initial work done for my alternative #7010 in a way that doesn't require any changes to the database drivers and works for all column types.

I agree that breaking changes shouldn't be a big deal but right now there don't appear to be any plans to release the next major version so we are stuck with what we have.

Reply:
I believe that the calculated fields should not be part of the main entity and that is why I think that your solution is poorly focused. A calculated field is a field that appears circumstantially in a query, but does not belong to the entity. That's my opinion.